traitsui/extras/demo.py

```diff
@@ -539,7 +541,7 @@ def _get_description(self):
         if self._description is None:
             self._get_init()
 
-        return self._description
+        return publish_html_str(self._description)
```

Comment:
This is an expensive operation, so we should use a `@cached_property` rather than regenerating every time.

Reply:
Fixed in 36adae8